cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/IndexTest.scala

```diff
@@ -185,9 +185,32 @@ MATCH (n:Person) WHERE n.name = $$value
 RETURN n
 ###
 
-An index can be automatically used for the equality comparison.
+An BTREE index can be automatically used for the equality comparison.
```

Comment:
```suggestion
A BTREE index can be automatically used for the equality comparison.
```

Reply:
Should index types have code formatting?
```suggestion
A `BTREE` index can be automatically used for the equality comparison.
```